contracts/src/interfaces/IGateway.sol

```diff
@@ -105,4 +105,7 @@ interface IGateway {
         uint128 destinationFee,
         uint128 amount
     ) external payable;
+
```

Comment:
Similar to the APIs for `registerToken` and `sendToken`, we need a `quoteTransactFee` function that calculates the amount of ether the user must provide when calling `transact`.

Reply:
https://github.com/Snowfork/snowbridge/pull/1141/commits/c192361a3bd28513f381d683f13192773b9fe6ad